daemon/cmd/cells.go

```diff
@@ -49,6 +54,9 @@ var (
 
 		// daemonCell wraps the legacy daemon initialization and provides Promise[*Daemon].
 		daemonCell,
+
+		// The BGP Control Plane
```

Comment:
I would expand this comment to have few more details (max 1-2 lines) and then document in bgpv1.Cell 
in more detail what it does (e.g. listens for these and these k8s events and talks to BGP in this way).